WordPress/Classes/Utility/OptimizelyHelper.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import Optimizely
+
+
+// MARK: - Define live variables used in Optimizely experiments.
+internal var optimizelyEnableNewSigninFlowKey: OptimizelyVariableKey = OptimizelyVariableKey.optimizelyKeyWithKey("EnableNewSigninFlow", defaultBOOL: false)
+
+
+/// A helper class for interacting with the Optimizely SDK
+///
+@objc class OptimizelyHelper: NSObject
+{
+
+    /// Activate Optimizely. Call this once when the application finishes launching.
+    ///
+    /// - Parameters:
+    ///     - launchOptions: The launchOptions dictionary from the app delegate.
+    ///
+    class func setupOptimizelyWithLaunchOptions(launchOptions: [NSObject: AnyObject]) {
+        Optimizely.sharedInstance().verboseLogging = true
```

Comment:
I can understand why this would be wanted when debugging, but I don't believe this should be in production.